README.md

```diff
@@ -313,6 +333,43 @@ Replace `${ACCESS_KEY}` with yours obtained from [Picovoice Console](https://con
 
 When done, release the resources allocated to Cobra using `cobra.release()`.
 
+### NodeJS
+
+Install NodeJS SDK:
+
+```console
+yarn add @picovoice/cobra-node
+```
+
+Create instances of the Cobra class:
+
+```javascript
+const { Cobra } = require("@picovoice/cobra-node");
+
+const accessKey = "${ACCESS_KEY}"; // Obtained from the Picovoice Console (https://console.picovoice.ai/)
+const handle = new Cobra(accessKey);
+```
+
+When instantiated, `handle` can process audio via its `.process` method.
+
+```javascript
+function getNextAudioFrame() {
+  // ...
+  return audioFrame;
+}
+
+while (true) {
+  const audioFrame = getNextAudioFrame();
+  const voiceProbability = handle.process(audioFrame);
+  console.log(voiceProbability);
+}
+```
+
+When done be sure to release resources using `release()`:
+
+```javascript
+handle.release();
```

Comment:
I feel like we should only use `handle` as a variable name in C. Otherwise it's a bit too low-level for your average user.